lxd/api_internal.go

```diff
@@ -241,6 +241,8 @@ func internalWaitReady(d *Daemon, r *http.Request) response.Response {
 	}
 
 	timeoutSeconds := request.QueryParam(r, "timeout")
+	waitNetwork := request.QueryParam(r, "network")
+	waitStorage := request.QueryParam(r, "storage")
```

Comment:
We're only acting on those network/storage params if the timeout param was provided. Is that intentional? What it I'd like to wait indefinitely for everything to be ready?